### DISCUSSION TITLE
File name conventions

### DISCUSSION THREAD

**Discussion by azkaafx** [gifaldyazkaa/kogasa-dscbot#179] [Q&A] [answered]
Hello, i just want to notice something that may does not important, but affects coding style of this project. It's conventions

This Repository uses TypeScript as core and main language, so this repository should be following TypeScript conventions style especially on file naming.

Is there any fixes for this simple thing?

Thanks in advance. 🙏🏻👍🏻

**Comment by azukashi** [comment, marked as answer]
That's right. TypeScript uses `PascalCase` and i just used random and unknown conventions style. I will starting to fix this!
